capi/VideoDecoderCapi.cpp

```diff
@@ -18,8 +18,8 @@
 #include "config.h"
 #endif
 #include "VideoDecoderCapi.h"
-#include "interface/VideoDecoderInterface.h"
-#include "interface/VideoDecoderHost.h"
+#include "VideoDecoderInterface.h"
+#include "VideoDecoderHost.h"
```

Comment:
Since interface not available outside the project, they need to be removed from the headers that get installed.  You can't have installed header file going into /usr/local/include that reference the interface directory.  According the the .pc file, the -I that projects should use is /usr/local/include/libyami or changed if --prefix is used.
If you just create a test program that uses the capi you will see the issues.